README.md

```diff
@@ -1,223 +1,67 @@
+[![GitHub release](https://img.shields.io/github/v/release/simplex-chat/simplexmq)](https://github.com/simplex-chat/simplexmq/releases)
 
-## Federated chat - private, secure, decentralised
+## Message broker for unidirectional (simplex) queues
 
-See [simplex.chat](https://simplex.chat) website for chat demo and the explanations of the system and how SMP protocol works.
+SimpleXMQ is a message broker for managing message queues and sending messages over public network. It consists of SMP server, SMP client library and SMP agent that implement [SMP protocol](./protocol/simplex-messaging.md) for client-server communication and [SMP agent protocol](./protocol/agent-protocol.md) to manage duplex connections via simplex queues on multiple SMP servers.
 
-SMP protocol is semi-formally defined [here](https://github.com/simplex-chat/protocol).
+SMP protocol is inspired by [Redis serialization protocol](https://redis.io/topics/protocol), but it is much simpler - it currently has only 8 client commands and 6 server responses.
 
-Currently only these features are available:
-- simple 1-to-1 chat with multiple people in the same terminal window.
-- auto-populated recipient name - just type your messages.
-- default server is available to play with - `smp.simplex.im:5223` - and you can deploy your own (`smp-server` executable in this repo).
-- no global identity or names visible to the server(s) - for the privacy of contacts and conversations.
-- E2E encryption, with public key that has to be passed out-of-band (see below)
-- authentication of each command/message with automatically generated RSA key pairs, separate for each conversation, the keys are not used as identity (2048 bit keys are used, it can be changed in [code via rsaKeySize setting](https://github.com/simplex-chat/simplex-messaging/blob/master/apps/dog-food/Main.hs))
+SimpleXMQ is implemented in Haskell - it benefits from robust software transactional memory and concurrency primitives that Haskell provides.
 
-Limitations/disclaimers:
-- no support for chat groups. It is coming in the next major version (i.e., not very soon:)
-- no delivery notifications - coming soon
-- no TCP transport encryption - coming soon (messages are encrypted e2e though, only random connection IDs and server commands are visible, but not the contents of the message)
-- system and protocol security was not audited yet, so you probably should NOT use it yet for high security communications - unless you know what you are doing.
+## SimpleXMQ roadmap
 
-## How to run chat client locally
+- Streams - high performance message queues. See [Streams RFC](./rfcs/2021-02-28-streams.md) for details.
+- "Small" connection groups, when each message will be sent by the SMP agent to multiple connections with a single client command. See [Groups RFC](./rfcs/2021-03-18-groups.md) for details.
+- SMP agents cluster to share connections and message management by multiple agents.
```

Comment:
I had to spend some time processing this - translate for people it would allow multi-device for simplex-chat?

Reply:
I understand it's only a subset of possible uses, but still

Reply:
Yes